app/controllers/sign_up/registrations_controller.rb

```diff
@@ -18,6 +18,7 @@ def new
       @register_user_email_form = RegisterUserEmailForm.new
       session[:sign_up_init] = true
       analytics.track_event(Analytics::USER_REGISTRATION_ENTER_EMAIL_VISIT)
+      render :new, locals: { request_id: nil }
```

Comment:
Where does `nil.to_s` come from? I want to pass in a nil value to the form. When the form renders, the `request_id` hidden input field won't have a value at all.